tests/unit/test_quant.py

```diff
@@ -29,36 +28,33 @@ def random_docs():
         yield d
 
 
-def get_output(req):
-    np.random.seed(531)
-
-    err = 0
-    for d in req.docs:
-        for c in d.chunks:
-            recv = pb2array(c.embedding)
-            send = np.random.random([embed_dim])
-            err += np.sum(np.abs(recv - send)) / embed_dim
-
-    print(f'reconstruction error: {err / num_docs:.6f}')
+def test_quant():
+    def get_output(req):
```

Comment:
Just out of interest: Why did you make this an inner function instead of a top level function as `random_docs()`?

Reply:
I just like to keep it inside a test to have it explicit that is related to this test.

Reply:
Shouldn't the `random_docs` then also be inside? It just feels like it is not consistent, at least for this test.